sdk/src/test/java/com/deploygate/sdk/HostAppTest.java

```diff
@@ -26,22 +26,63 @@ public void setUp() {
 
     @Test
     public void default_properties() {
-        HostApp app = new HostApp(context);
+        HostApp app = new HostApp(
+                context,
+                new DeployGateSdkConfiguration.Builder().build()
+        );
 
-        Truth.assertThat(app.debuggable).isTrue();
         Truth.assertThat(app.canUseLogcat).isTrue();
         Truth.assertThat(app.packageName).isEqualTo("com.deploygate.sdk.test");
         Truth.assertThat(app.sdkVersion).isEqualTo(4);
         Truth.assertThat(app.sdkArtifactVersion).isEqualTo("4.6.1");
+        Truth.assertThat(app.activeFeatureFlags).isEqualTo(31);
```

Comment:
I understood this assertion expected `31` because  activeFeatureFlags has 0x11111 -> 31 when all features enabled.